apps/webClient/src/components/NavButton/index.tsx

```diff
@@ -1,43 +1,65 @@
 import React from "react";
+import clsx from "clsx";
 import { makeStyles } from "@material-ui/styles";
 import Button from "@material-ui/core/Button";
-import Typography from "@material-ui/core/Typography";
 import { Theme } from "@material-ui/core/styles/createMuiTheme";
 
 interface Props {
   route: string;
   text: string;
+  vertical?: boolean;
+  isLeft?: boolean;
 }
 
 const useStyles = makeStyles((theme: Theme) => ({
-  ourWork: {
-    backgroundColor: "white",
+  verticalBtn: {
+    backgroundColor: "transparent",
     borderRadius: 0,
+    border: "solid 3px white",
     width: 60,
     height: 200,
+    padding: "30px 10px",
+    textTransform: "uppercase",
   },
-  aboutUs: {
+  horizontalBtn: {
     backgroundColor: "transparent",
     borderRadius: 0,
-    border: "solid 4px white",
-    width: 60,
-    height: 200,
+    border: "solid 3px white",
+    width: 200,
+    height: 60,
+    textTransform: "uppercase",
+    fontSize: 18,
   },
-  buttonLabel: {
+  verticalLabel: {
     writingMode: "vertical-rl",
     textOrientation: "sideways",
-    fontSize: 20,
-    textTransform: "uppercase",
+    fontSize: 18,
+  },
+  verticalLabelLeft: {
+    transform: "rotate(-180deg)",
   },
 }));
 
-const NavButton: React.FC<Props> = ({ route, text }) => {
+const NavButton: React.FC<Props> = ({
+  route,
+  text,
+  vertical = false,
+  isLeft = false,
+}) => {
   const classes = useStyles();
+
+  const deriveButtonLabel = (vertical: boolean, isLeft: boolean) => {
+    if (!vertical) return {};
+    const labelOverride = [classes.verticalLabel];
+    if (isLeft) labelOverride.push(classes.verticalLabelLeft);
+    return { label: clsx(labelOverride) };
+  };
```

Comment:
nitpick, no need to change, but more verbosity in naming stuff goes a long way -- `deriveButtonLabel` reads like we are trying to get the label for the button.

`deriveButtonClassOverrides` helps the next person reading the code to more quickly understand what's going on.

Back to the self review :) -- generally, these sorts of things are improvements I make as I do a self review.  Yes, it slows us down in the near term but extra readable code helps whomever looks at this in the future.